test/testUtil.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * testUtil.c
+ *
+ * contains the functions necessary for the testing framework
+ */
+
+#ifndef TEST_UTIL_H_
+#define TEST_UTIL_H_
+
+#include <stdlib.h>
+#include <stdio.h>
+
+#define assertTrue(cond) \
```

Comment:
Should this be defined as `assert_true`, using snake case, just like all of the other functions in the source code? Please be consistent.